demosplan/DemosPlanCoreBundle/Logic/Factory/ParagraphFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
```

Comment:
I would vote to put this method into the existing ```demosplan/DemosPlanCoreBundle/Logic/Document/ParagraphHandler``` and name it like the one above - like ```administrationDocumentDeleteHandler```

Then implement a ```ParagraphHandlerInterface``` and put that into demosplan-addon

Reply:
done in [3fe7b86](https://github.com/demos-europe/demosplan-core/pull/3290/commits/3fe7b8642fce0c0ba7708864973c587d83b6be55)